StRoot/StGmtAligner/StGmtAligner.cxx

```diff
@@ -0,0 +1,150 @@
+#include "StGmtAligner.h"
+#include <stdlib.h>
+#include "Riostream.h"
+#include "TROOT.h"
+#include "TSystem.h"
+#include "TFile.h"
+#include "TKey.h"
+#include "TRandom.h"
+#include "TTree.h"
+#include "TBranch.h"
+#include "TStopwatch.h"
+#include "StThreeVectorF.hh"
+#include "StMatrixF.hh"
+#include "TH1.h"
+#include "TH2.h"
+#include "TProfile.h"
+#include "TMath.h"
+#include "TVector3.h"
+#include "TProcessID.h"
+#include "StEvent.h"
+#include "StPrimaryVertex.h"
+#include "StBFChain.h"
+#include "TGeoMatrix.h"
+#include "EventT.h"
+#include "StTpcDb/StTpcDb.h"
+#include "StDetectorDbMaker/StGmtSurveyC.h"
+//________________________________________________________________________________
+StGmtAligner::StGmtAligner(const Char_t *name) : StMaker(name),fFile(0), fTree(0), fEvent(0) {
```

Comment:
StGmtAligner is a maker.  The "maker" rules should apply here.

1. The name of the code should by in the form StTLA<Blah>Maker.  e.g. StGmtAlignmentMaker
2. The code should be organized under StRoot/StGmtAlignmentMaker/

Reply:
Done